### PR TITLE
Fixed #3

### LINKED ISSUE #3: please add a GPT generated section about inflation

### PR DESCRIPTION
**Pull request by HannesDiemerling**

### CODE REVIEW

File: inflation.Rmd [hunk 1]
```diff
@@ -115,3 +115,9 @@ inflation %>%
   geom_line(aes(date, value, color = name)) +
   NULL
 ```
+
+Inflation is an economic phenomenon characterized by a rise in the general price level of goods and services within an economy over a period of time. 
+When inflation occurs, each unit of currency buys fewer goods and services, indicating a loss of purchasing power. 
+It's usually measured by the Consumer Price Index (CPI) or the Producer Price Index (PPI).
```
Comment (aaronpeikert) [inline inflation.Rmd:121]: Are you sure we have to mention this?

File: inflation.Rmd [hunk 2]
```diff
@@ -115,3 +115,9 @@ inflation %>%
   geom_line(aes(date, value, color = name)) +
   NULL
 ```
+
+Inflation is an economic phenomenon characterized by a rise in the general price level of goods and services within an economy over a period of time. 
+When inflation occurs, each unit of currency buys fewer goods and services, indicating a loss of purchasing power.
```
Comment (aaronpeikert) [inline inflation.Rmd]:
```suggestion
When inflation happens, each unit of currency buys fewer goods and services, indicating a loss of purchasing power. 
```

**Review by aaronpeikert — CHANGES_REQUESTED** [review]

File: inflation.Rmd (hunk 1 above)
Comment (HannesDiemerling) [reply inflation.Rmd:121]: i think this is a important part.

**Review by aaronpeikert — APPROVED** [review]
This looks amazing!

**Pull request merged**